core/rpc/src/author/error.rs

```diff
@@ -63,6 +63,41 @@ impl From<Error> for rpc::Error {
 				message: e.description().into(),
 				data: Some(format!("{:?}", e).into()),
 			},
+			Error(ErrorKind::Pool(txpool::error::ErrorKind::InvalidTransaction(code)), _) => rpc::Error {
+				code: rpc::ErrorCode::ServerError(ERROR + 3),
```

Comment:
Are these codes used for something?

Reply:
They can be arbitrary and it's up to us to define their meaning. They should just be unique, I suppose they are going to be useful for the consumers of the API.

Reply:
If they are useful for the consumers of the API, wouldn't it be better to put them into constants?

Reply:
Updated

Reply:
Ty